### DISCUSSION TITLE
Option to rename skins in lazer

### DISCUSSION THREAD

**Discussion by mixalisnew** [ppy/osu#26577] [Ideas] [answered]
As of now in osu stable we can go the game files and change the skin folder name to what ever we want etc DT blue, Whitecat v3 or whatever. In lazer we cant do that and the skin names can be really messy. I believe an option to right click on the skin and have the option to rename it would help organize them. 

Thanks for your time

**Comment by Gabixel** [comment]
Looks like a duplicate of https://github.com/ppy/osu/discussions/21231 (not for the described scenario, but because they share the same idea: renaming a skin).
The skin editor will definitely be improved with more features (this one included since it seems in high demand)!

**Comment by mixalisnew** [comment, marked as answer]
ITS FINALLY DONE!!! i can rest in peace.
